components/clientComponents/globals/Header/YourAccountDropdown.tsx

```diff
@@ -64,7 +64,7 @@ export const YourAccountDropdown = ({ isAuthenticated }: YourAccountDropdownProp
               >
                 <DropdownMenuItem href={`/${i18n.language}/profile`} text={t("adminNav.profile")} />
 
-                {(ability?.can("view", "Flag") || ability?.can("view", "User", "Privilege")) && (
+                {(ability?.can("view", "Flag") || ability?.can("view", "Privilege")) && (
```

Comment:
Correcting a syntax error that was providing the desired effect but not technically correct.
`ability.can("view", "User", "Privilege");` is verifying if a user can `view` the `privilege` field on a `user` object.